man/systemd.unit.xml

```diff
@@ -56,7 +56,7 @@
 
     <refsect2>
       <title>User Unit Search Path</title>
-      <para><literallayout><filename>~/.config/systemd/user.control/*</filename>
+      <para><literallayout><filename>$XDG_CONFIG_HOME/systemd/user.control/*</filename>
```

Comment:
tbh I'd really drop XDG_CONFIG_HOME from the docs. It's confusing enough as it is. Where we don't need the indirections we should not confuse people with it, in particular as people might then think it's actually a good idea to use thatenv var and redirect things. I'd just show the literal path everywhere, even if we internally use the env var